packages/cli/lib/prompts/projectAddPrompt.js

```diff
@@ -0,0 +1,55 @@
+const { promptUser } = require('./promptUtils');
+const { fetchJsonFromRepository } = require('@hubspot/cli-lib/github');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+
+const i18nKey = 'cli.lib.prompts.projectAddPrompt';
+
+const createTypeOptions = async () => {
+  const config = await fetchJsonFromRepository(
+    'hubspot-project-components',
+    'main/config.json'
+  );
+  return config.components;
```

Comment:
This is a similar suggestion to the one I had above. It's related to keeping the CLI blissfully unaware of any of the specifics of the config file in the `hubspot-project-components` repo. 

What if we could achieve the same functionality here, but in a more generic way without requiring a `components` key in the repo that the CLI needs to be aware of.

**Consider a flow along the lines of this:**
(This is just an idea to consider)

the repo is structured something like this:
```
config.json
components/
  config.json
  crm-card.js
  serverless-function.js
  app/
```
And the two configs look something along the lines of:
top level config
```json
{ "path": "components", "label": "Component" }
```
nested config
```json
{ "path": "app", "label": "App", "insertPath": "./", "isFile": false, "type": "PRIVATE_APP" }
```
This would enable us to run a command along the lines of `hs project add` without specifying any component or asset type. Then it would read from the top level config and use that to generate the prompt options for the user to select from. Once they select an option, you dive into the folder that they selected and you read the next config file and use that to dynamically generate the next prompt (and so on until the user selects an option that contains `insertPath`).

We would still support things like `hs project add component` because the command could be smart enough to know that it needs to immediately look inside of the "components" folder.